hydrolib/core/dflowfm/polyfile/models.py

```diff
@@ -106,7 +129,53 @@ def _get_serializer(cls) -> Callable:
 
     @classmethod
     def _get_parser(cls) -> Callable:
-        # TODO Prevent circular dependency in Parser
+        # Prevent circular dependency in Parser
         from .parser import read_polyfile
 
         return read_polyfile
+
+    @property
+    def x(self) -> List[float]:
```

Comment:
What is this used for? You typically want to keep the individual PolyObjects separated.

Reply:
It is used in the `SourceSinkConverter.convert` to to get the xcoordinates